extension-spring/src/main/java/io/smallrye/openapi/spring/ParameterProcessor.java

```diff
@@ -1036,6 +1034,11 @@ static String pathOf(AnnotationTarget target) {
                     path = methodAnnotation;
                 }
             }
+            // Also support @RequestMapping
+            AnnotationInstance methodAnnotation = target.asMethod().annotation(SpringConstants.REQUEST_MAPPING);
+            if (methodAnnotation != null && methodAnnotation.value() != null) {
+                path = methodAnnotation;
+            }
```

Comment:
Should this code also apply to classes with the `@RequestMapping` annotation?

Reply:
Y. @RequestMapping is a valid class-level annotation. Not sure that needs to be on this pass..

Reply:
Yes maybe. We should create a test case for that then ...